test/test_images/probe_helper/probe_helper_test.go

```diff
@@ -284,6 +315,9 @@ func TestProbeHelper(t *testing.T) {
 	// Run the test CloudStorageSource.
 	runTestCloudStorageSource(ctx, gotCloudStorageRequest, receiverURL)
 
+	// Run the test CloudSchedulerSource.
+	runTestCloudSchedulerSource(ctx, time.Second, receiverURL)
```

Comment:
nit: set a shorter period to speed up the test.

Reply:
Done.